djreact/djangoapps/trips/managers.py

```diff
@@ -17,6 +17,24 @@ def get_queryset(self):
         Usage:
             >>> TripSchedule.available.all()
         """
-        return super(AvailableTripsManager, self).get_queryset().filter(
+        return super(AvailableTripScheduleManager, self).get_queryset().filter(
```

Comment:
For a trip, will we be getting the active schedules by trip.trip_schedules.available.all()?

Reply:
You have raised a very good question, Which I would love to clarify. 

For getting available schedules you would
`trip.trip_schedule(manager='available').all()`
and for getting all the schedule

`trip.trip_schedule.all()`

Reply:
cool. Thanks for the explanation.